src/OSPSuite.UI/Views/Importer/MetaDataParameterEditorView.cs

```diff
@@ -0,0 +1,25 @@
+﻿using OSPSuite.Assets;
+using OSPSuite.Presentation.Presenters.Importer;
+using OSPSuite.Presentation.Views.Importer;
+using OSPSuite.UI.Controls;
+
+namespace OSPSuite.UI.Views.Importer
+{
+   public partial class MetaDataParameterEditorView : BaseUserControl, IMetaDataParameterEditorView
+   {
+      private IMetaDataParameterEditorPresenter _presenter;
+
+      public MetaDataParameterEditorView()
+      {
+         InitializeComponent();
+         this.manualnputLayoutControlItem.Text = Captions.Importer.ManualInput;
```

Comment:
DO this in the InitializeResources Method

Reply:
done